SPECS/plexus-sec-dispatcher/plexus-sec-dispatcher.spec

```diff
@@ -34,24 +32,30 @@ Plexus Security Dispatcher Component
 %{?javadoc_package}
  
 %prep
-%setup -q
-%patch 0 -p1
- 
+%autosetup -n %{name}-%{name}-%{version}
+
 cp %{SOURCE1} .
  
 %pom_remove_parent
-%pom_xpath_inject "pom:dependency[pom:artifactId='junit']" "<scope>test</scope>"
+%pom_xpath_inject 'pom:project' '<groupId>org.codehaus.plexus</groupId>'
  
 %mvn_file : plexus/%{name}
+
+%mvn_alias org.codehaus.plexus: org.sonatype.plexus:
  
 %build
-%mvn_build -- -Dmaven.compiler.source=1.8 -Dmaven.compiler.target=1.8 
+%mvn_build -- -Dmaven.compiler.source=1.8 -Dmaven.compiler.target=1.8
+
 %install
 %mvn_install
-%files -n %{?module_prefix}%{name} -f .mfiles
+
+%files -f .mfiles
 %license LICENSE-2.0.txt
  
 %changelog
+* Mon Apr 01 2024 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 2-1
+- Auto-upgrade to 2.0 - 3.0 package upgrade
```

Comment:
in the changelog can you credit fedora if you used them. thanks

Reply:
Added. Thanks :)